recipes/dionysus/meta.yml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dionysus" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "78c24da06fd273f0103ad4cb86331f5f4518d7f0445e2cecd0e325ca1a2d011f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - cmake
+        - boost
```

Comment:
Since you're compiling C code here, you should use the `toolchain` package as a build dependency.